### PR TITLE
Refetch full content for generic show recordings

### PR DESCRIPTION
**Pull request by jeremy**
## Summary

- `basecamp show <id>` without a type uses the `/recordings/` endpoint, which returns sparse data (no content body)
- Now re-fetches using the type-specific endpoint derived from the response's canonical API `type` field
- Maps SDK `RecordingType` constants to endpoints: `Kanban::Card` → `/card_tables/cards/`, `Schedule::Entry` → `/schedule_entries/`, `Vault::Document` → `/documents/`, etc.
- Falls through gracefully to sparse recording data for unrecognized types (no regression)
- The refetch endpoint is derived from the type field, **never from the url field**, which could point off-origin and leak the bearer token

## Test plan

- [x] `basecamp show <message-id>` → full message content (verified with explicit type and via URL)
- [x] `basecamp show <todo-id>` → full todo content (verified with explicit type and via URL)
- [x] `basecamp show <card-id>` → full card content (Kanban::Card mapping, verified with explicit type)
- [x] `basecamp show <schedule-entry-id>` → full schedule entry — no schedule entries in test accounts; endpoint mapping verified by unit test
- [x] `basecamp show <nonexistent-id>` → clean error (exit 2, structured JSON)
- [x] `bin/ci` passes
- [x] New unit tests: `TestRecordingTypeEndpoint*` (14 type cases + 3 edge cases)
- [x] New integration tests: `TestShowGenericRecordingRefetchesTypeEndpoint` (verifies 2nd GET to type endpoint) and `TestShowGenericRecordingFallsBackOnRefetchError` (verifies graceful fallback)

### CODE REVIEW

File: internal/commands/show.go [hunk 1]
```diff
@@ -161,6 +178,39 @@ You can also pass a Basecamp URL directly:
...
+	case "Comment":
+		return fmt.Sprintf("/comments/%s.json", id)
+	case "Kanban::Card":
+		return fmt.Sprintf("/card_tables/cards/%s.json", id)
+	case "Document", "Vault::Document":
+		return fmt.Sprintf("/documents/%s.json", id)
+	case "Schedule::Entry":
+		return fmt.Sprintf("/schedule_entries/%s.json", id)
+	case "Question::Answer":
+		return fmt.Sprintf("/questions/%s.json", id)
```
[the remark this answers is not in the document]
Comment (jeremy) [reply internal/commands/show.go:204]: Fixed — `Question::Answer` now maps to `/question_answers/` (3b98d31), and `Question` → `/questions/` added in c52397a.

File: internal/commands/show.go [hunk 2]
```diff
@@ -161,6 +178,39 @@ You can also pass a Basecamp URL directly:
...
+	case "Inbox::Forward":
+		return fmt.Sprintf("/forwards/%s.json", id)
+	case "Upload":
+		return fmt.Sprintf("/uploads/%s.json", id)
+	default:
+		return ""
+	}
+}
```
[the remark this answers is not in the document]
Comment (jeremy) [reply internal/commands/show.go:214]: Fixed — added `Question` → `/questions/` and `Todolist::Todo` → `/todos/` cases with tests in c52397a.

File: internal/commands/show_test.go [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+	}{
+		{"Todo", "123", "/todos/123.json"},
+		{"Todolist", "456", "/todolists/456.json"},
+		{"Message", "789", "/messages/789.json"},
+		{"Comment", "100", "/comments/100.json"},
+		{"Kanban::Card", "200", "/card_tables/cards/200.json"},
+		{"Document", "300", "/documents/300.json"},
+		{"Vault::Document", "301", "/documents/301.json"},
+		{"Schedule::Entry", "400", "/schedule_entries/400.json"},
+		{"Question::Answer", "500", "/questions/500.json"},
```
[the remark this answers is not in the document]
Comment (jeremy) [reply internal/commands/show_test.go]: Fixed — test now includes `Question` and `Todolist::Todo` cases. See c52397a.

File: internal/commands/show.go [hunk 4]
```diff
@@ -161,6 +178,39 @@ You can also pass a Basecamp URL directly:
...
+// type-specific endpoint path. Type names are the namespaced forms returned by
+// the Basecamp API (e.g. "Kanban::Card", "Schedule::Entry"), matching the SDK
+// constants in basecamp.RecordingType*. Returns "" for unrecognized types,
+// causing the caller to fall through to sparse recording data (no regression).
+func recordingTypeEndpoint(data map[string]any, id string) string {
+	t, _ := data["type"].(string)
+	switch t {
+	case "Todo":
+		return fmt.Sprintf("/todos/%s.json", id)
+	case "Todolist":
```
[the remark this answers is not in the document]
Comment (jeremy) [reply internal/commands/show.go]: Fixed — `Todolist::Todo` now maps to `/todos/` in c52397a.

File: internal/commands/show.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (jeremy) [reply internal/commands/show.go]: Fixed — `Question::Answer` corrected to `/question_answers/` in 3b98d31.

File: internal/commands/show.go [hunk 5]
```diff
@@ -123,6 +123,23 @@ You can also pass a Basecamp URL directly:
...
 
+			// For generic recording lookups, re-fetch using the type-specific
+			// endpoint to get full content (the /recordings/ endpoint returns
+			// sparse data). The endpoint is derived from the response's type
+			// field — never from the url field, which could point off-origin.
+			if recordType == "" || recordType == "recording" || recordType == "recordings" {
+				if refetchEndpoint := recordingTypeEndpoint(data, id); refetchEndpoint != "" {
+					refetchResp, refetchErr := app.Account().Get(cmd.Context(), refetchEndpoint)
+					if refetchErr == nil && refetchResp.StatusCode != http.StatusNoContent {
+						var richer map[string]any
```
[the remark this answers is not in the document]
Comment (jeremy) [reply internal/commands/show.go:134]: Fixed — added `TestShowGenericRecordingRefetchesTypeEndpoint` (verifies 2 requests: /recordings/ then /todos/) and `TestShowGenericRecordingFallsBackOnRefetchError` (verifies sparse data fallback on refetch failure). See 67e8e04.
